rules/integrations/azure/privilege_escalation_azure_kubernetes_rolebinding_modified_or_deleted.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the modification or deletion of potential malicious rolebinding. You can assign these roles to Kubernetes subjects 
+(users, groups, or service accounts) with role bindings and cluster role bindings. An adversary who has permissions to create 
+bindings and cluster-bindings in the cluster can create a binding to the cluster-admin ClusterRole or to other high privileges roles.
```

Comment:
There is no check to differentiate common and potential malicious. Also, `An adversary who has permissions to create 
bindings and cluster-bindings in the cluster can create a binding to the cluster-admin ClusterRole or to other high privileges roles.` this would be an creation event, not modification/deletion